### ISSUE TITLE
FDSN down 07/12/2016

### ISSUE THREAD

**Issue by calum-chamberlain** [GeoNet/help#29]
Hello all,

I'm unable to download from the GeoNet FDSN service (via obpsy) this morning and overnight.  I guess and hope that this is temporary?

```python
from obspy.clients.fdsn import Client
client = Client('GEONET')
---------------------------------------------------------------------------
FDSNException                             Traceback (most recent call last)
<ipython-input-3-0dc335636bf5> in <module>()
----> 1 client = Client('GEONET')

/home/calumch/my_programs/Python_virtual_envs/eq_2/local/lib/python2.7/site-packages/obspy/clients/fdsn/client.pyc in __init__(self, base_url, major_versions, user, password, user_agent, debug, timeout, service_mappings, force_redirect)
    256             print("Request Headers: %s" % str(self.request_headers))
    257 
--> 258         self._discover_services()
    259 
    260     def get_events(self, starttime=None, endtime=None, minlatitude=None,

/home/calumch/my_programs/Python_virtual_envs/eq_2/local/lib/python2.7/site-packages/obspy/clients/fdsn/client.pyc in _discover_services(self)
   1480                    "be due to a temporary service outage or an invalid FDSN "
   1481                    "service address." % self.base_url)
-> 1482             raise FDSNException(msg)
   1483 
   1484         # Cache.

FDSNException: No FDSN services could be discovered at 'http://service.geonet.org.nz'. This could be due to a temporary service outage or an invalid FDSN service address.
```

**Comment by calum-chamberlain** [comment]
Looks like it's up again - out of curiosity, is this something that we can expect often, and is there an obvious cause? I have seen it go down quite often (but not reported it because it comes back), but it ends up failing tests on EQcorrscan that rely on downloading data.  It also means that I can't rely on running aftershock detections etc downloading data on-the-fly...

**Comment by nbalfour** [comment]
Hi Calum, 

Yes, FDSN went down last night and has been restarted. 

We are currently in a bit of a transition and GeoNet is in the process of appointing a new Data Management Team Leader. This person will be responsible for reviewing our data services and the associated support levels. 

We have different support levels (aka tiers) for our services and they are monitored/repaired according. 

Unfortunately, FDSN currently doesn't have a support level defined but as I mentioned this will change with the incoming Data Manager. 

Please keep in mind that anything built on GeoNet data services will only be as reliable as the support level/tier for that service.

So, in conclusion, yes, we understand this is a problem and it will be looked at in more detail in near future. 

Thank you for bring the problem to our attention.
